tests/network/general/test_bridge_marker.py

```diff
@@ -127,11 +127,13 @@ def non_homogenous_bridges(worker_node1, worker_node2):
             yield bridgemarker2_ncp, bridgemarker3_ncp
 
 
-def _assert_failure_reason_is_bridge_missing(pod, bridge):
-    cond = pod.instance.status.conditions[0]
-    missing_resource = bridge.resource_name
-    assert cond.reason == "Unschedulable"
-    assert f"Insufficient {missing_resource}" in cond.message
+def _assert_failure_reason_is_bridge_missing(vmi, bridge):
+    for cond in vmi.instance.status.conditions:
+        if cond.type == "PodScheduled":
+            assert cond.status == "False" and cond.reason == "Unschedulable"
```

Comment:
In case one fails, it will be hard to detect which one.
I suggest you add multiple assertions or a single one that includes all in parallel (unsure if the later can be done easily with pytest).